### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by ghost**
Some README improvements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Usage: npm <command> blah blah...
 ```
 
 ## One more thing
-You can notice good news when you didn't know that npm latest version was updated.
+npmbrew uses [update-notifier](https://github.com/yeoman/update-notifier/), to let users know when there is a new version available for download.
```
Comment (watilde) [inline README.md:69]: npmbrew use update-notifier for oneself. Display update notifier for npm by [this func](https://github.com/watilde/npmbrew/blob/master/lib/modules/npm.js#L44-L64). So could you changed this description only a little?

**Comment by watilde** [comment]
Amazing! I commented a little one, after fixed that I'd like to land this.

Thanks always.

File: README.md (hunk 1 above)
Comment (ghost) [inline README.md:69]: You use it on a couple of places actually, one of them is `cli.js`:

``` js
updateNotifier({
  packageName : pkg.name,
  packageVersion : pkg.version
}).notify()
```

This is pure `update-notifier`.

Anyhow, how would you like to phrase it?

Comment (watilde) [inline README.md:69]: Oh certainly it was so! That's correct.

**Comment by watilde** [comment]
I'll npm version patch, Thanks!

File: README.md (hunk 1 above)
Comment (ghost) [inline README.md:69]: I believe you are using this package, for (2) things:
- Notify users about npmbrew updates.
- Notify users about npm updates.

If this is correct, then the README is OK, but we could certainly add more information later.

**Comment by ghost** [comment]
Thanks! What do you think about integrating `commander.js` as well?

File: README.md (hunk 1 above)
Comment (watilde) [inline README.md:69]: That's correct!
- Also I used update-notifier both
- The latter one sould write at here

**Pull request merged**